examples/draw-geometry.js

```diff
@@ -0,0 +1,29 @@
+goog.require('ol.geom.LineString');
+goog.require('ol.geom.Point');
+goog.require('ol.geom.Polygon');
+goog.require('ol.render');
+goog.require('ol.style.Circle');
+goog.require('ol.style.Fill');
+goog.require('ol.style.Stroke');
+
+
+var canvas = document.getElementById('canvas');
+var pixelRatio = ol.has.DEVICE_PIXEL_RATIO;
+canvas['width'] = 100 * pixelRatio;
+canvas['height'] = 100 * pixelRatio;
+
+var render = ol.render.toContext(canvas.getContext('2d'), pixelRatio);
+
+var fill = new ol.style.Fill({ color: 'blue' });
+var stroke = new ol.style.Stroke({ color: 'black' });
+render.setFillStrokeStyle(fill, stroke);
+render.setImageStyle(new ol.style.Circle({
+  radius: 10,
+  fill: fill,
+  stroke: stroke
+}));
+
+render.drawLineStringGeometry(new ol.geom.LineString([[10, 10], [90, 90]]));
```

Comment:
Ah I guess this is handled automatically. Maybe it would be cleaner then if ol.render.toContext got a width and height parameter, so people do not have the apply the pixelRatio math there?